### PR TITLE
Update generate Debian package tool help output

### LINKED ISSUE #538: Missing option in debs help
Hello team, 

In the script to generate the package in debian, the option `--packages-branch <branch>` is missing in the help.

## Example:
- Debs: `./generate_debian_package.sh -h`
```bash
Usage: ./generate_debian_package.sh [OPTIONS]
    -b, --branch <branch>     [Required] Select Git branch []. By default: master.
    -t, --target <target>     [Required] Target package to build: manager, api or agent.
    -a, --architecture <arch> [Optional] Target architecture of the package [amd64/i386/ppc64le/arm64/armhf].
    -j, --jobs <number>       [Optional] Change number of parallel jobs when compiling the manager or agent. By default: 2.
    -r, --revision <rev>      [Optional] Package revision. By default: 1.
    -s, --store <path>        [Optional] Set the destination path of package. By default, an output folder will be created.
    -p, --path <path>         [Optional] Installation path for the package. By default: /var/ossec.
    -d, --debug               [Optional] Build the binaries with debug symbols. By default: no.
    -c, --checksum <path>     [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
    --dont-build-docker       [Optional] Locally built docker image will be used instead of generating a new one.
    --sources <path>          [Optional] Absolute path containing wazuh source code. This option will use local source code instead of downloading it from GitHub.
    --dev                     [Optional] Use the SPECS files stored in the host instead of downloading them from GitHub.
    --future                  [Optional] Build test future package x.30.0 Used for development purposes.
    -h, --help                Show this help.
```
Missing `--packages-branch` option.

It should be like the RPM's one: `./generate_rpm_package.sh -h`
```bash
Usage: ./generate_rpm_package.sh [OPTIONS]
    -b, --branch <branch>        [Required] Select Git branch or tag e.g. master
    -t, --target <target>        [Required] Target package to build [manager/api/agent].
    -a, --architecture <arch>    [Optional] Target architecture of the package [x86_64/i386/ppc64le/aarch64/armv7hl].
    -r, --revision <rev>         [Optional] Package revision that append to version e.g. x.x.x-rev
    -l, --legacy                 [Optional] Build package for CentOS 5.
    -s, --store <path>           [Optional] Set the destination path of package. By default, an output folder will be created.
    -j, --jobs <number>          [Optional] Number of parallel jobs when compiling.
    -p, --path <path>            [Optional] Installation path for the package. By default: /var/ossec.
    -d, --debug                  [Optional] Build the binaries with debug symbols and create debuginfo packages. By default: no.
    -c, --checksum <path>        [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
    --dont-build-docker          [Optional] Locally built docker image will be used instead of generating a new one.
    --sources <path>             [Optional] Absolute path containing wazuh source code. This option will use local source code instead of downloading it from GitHub.
    --packages-branch <branch>   [Optional] Select Git branch or tag from wazuh-packages repository. e.g master
    --dev                        [Optional] Use the SPECS files stored in the host instead of downloading them from GitHub.
    --src                        [Optional] Generate the source package in the destination directory.
    --future                     [Optional] Build test future package x.30.0 Used for development purposes.
    -h, --help                   Show this help.
```

Regards

### PR DESCRIPTION
**Pull request by jcruzlp**
|Related issue|
|---|
|#538|

## Description

Closes #538 

Added the `--packages-branch` option description to the `--help` option as shown in the issue.
Also added one space to the previous options to the new exit's text be aligned

In addition checked out the other helps options and find out that there's `-v` `--verbose` options on macos `generate_wazuh_packages.sh` and those don't appears on the `-h` prompt, so also added it.
## Logs example

Now using `./generate_debian_package.sh -h` shows the following prompt:

```
Usage: ./generate_debian_package.sh [OPTIONS]

    -b, --branch <branch>      [Required] Select Git branch []. By default: master.
    -t, --target <target>      [Required] Target package to build: manager, api or agent.
    -a, --architecture <arch>  [Optional] Target architecture of the package [amd64/i386/ppc64le/arm64/armhf].
    -j, --jobs <number>        [Optional] Change number of parallel jobs when compiling the manager or agent. By default: 2.
    -r, --revision <rev>       [Optional] Package revision. By default: 1.
    -s, --store <path>         [Optional] Set the destination path of package. By default, an output folder will be created.
    -p, --path <path>          [Optional] Installation path for the package. By default: /var/ossec.
    -d, --debug                [Optional] Build the binaries with debug symbols. By default: no.
    -c, --checksum <path>      [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
    --dont-build-docker        [Optional] Locally built docker image will be used instead of generating a new one.
    --sources <path>           [Optional] Absolute path containing wazuh source code. This option will use local source code instead of downloading it from GitHub.
    --packages-branch <branch> [Optional] Select Git branch or tag from wazuh-packages repository. e.g master.
    --dev                      [Optional] Use the SPECS files stored in the host instead of downloading them from GitHub.
    --future                   [Optional] Build test future package x.30.0 Used for development purposes.
    -h, --help                 Show this help.

```
And using `./generate_wazuh_packages.sh -h` shows:

```
Usage: ./generate_wazuh_packages.sh [OPTIONS]

  Build options:
    -b, --branch <branch>         [Required] Select Git branch or tag e.g. 
    -s, --store-path <path>       [Optional] Set the destination absolute path of package.
    -j, --jobs <number>           [Optional] Number of parallel jobs when compiling.
    -r, --revision <rev>          [Optional] Package revision that append to version e.g. x.x.x-rev
    -c, --checksum <path>         [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
    -h, --help                    [  Util  ] Show this help.
    -i, --install-deps            [  Util  ] Install build dependencies (Packages).
    -x, --install-xcode           [  Util  ] Install X-Code and brew. Can't be executed as root.
    -v, --verbose                 [  Util  ] Show additional information during the package generation.

  Signing options:
    --keychain                    [Optional] Keychain where the Certificates are installed.
    --keychain-password           [Optional] Password of the keychain.
    --application-certificate     [Optional] Apple Developer ID certificate name to sign Apps and binaries.
    --installer-certificate       [Optional] Apple Developer ID certificate name to sign pkg.
    --notarize                    [Optional] Notarize the package for its distribution on macOS Catalina .
    --developer-id                [Optional] Your Apple Developer ID.
    --altool-password             [Optional] Temporary password to use altool from Xcode.


```

### CODE REVIEW

File: macos/generate_wazuh_packages.sh [hunk 1]
```diff
@@ -185,6 +185,7 @@ function help() {
     echo "    -h, --help                    [  Util  ] Show this help."
     echo "    -i, --install-deps            [  Util  ] Install build dependencies (Packages)."
     echo "    -x, --install-xcode           [  Util  ] Install X-Code and brew. Can't be executed as root."
+    echo "    -v, --verbose                 [  Util  ] Show additional information during installation."
```
Comment (DFolchA) [inline macos/generate_wazuh_packages.sh:188]: This script does not install anything, it generates a package.

**Review by DFolchA — CHANGES_REQUESTED** [review]

File: macos/generate_wazuh_packages.sh (hunk 1 above)
Comment (jcruzlp) [reply macos/generate_wazuh_packages.sh]: Fixed here: ca4a97a3421ca8a4fde214fe596ac3af99de2806

**Review by okynos — APPROVED** [review]
LGTM.

**Review by DFolchA — APPROVED** [review]
LGTM

**Pull request merged**